hw4/02-sessions.js

```diff
@@ -1,17 +1,38 @@
+
 const express = require('express');
 const session = require('express-session');
-const app = express();
-const port = process.env.PORT || 5001;
+const app     = express();
+const port    = process.env.PORT || 5001;
+
 
-// Add your code here
+app.use(session({secret: 'secret_key'}));
 
-// Use the express-session module
-app.use(/** ... */);
 
 app.get('/', (req, res) => {
-  // Add your code here
+  if (req.session.first_time) {
+    req.session.routes.push('/');
```

Comment:
when you are have line 18 : req.session.routes = ['/'];
why do you need to push "/" inside the first if

Reply:
That if statement triggers if the user visits the homepage for the first time (localhost:5001/). I push it to store that they visited '/'.